bin/_test-run.sh

```diff
@@ -42,143 +42,143 @@ function upgrade_integration_tests() {
     cleanup
 }
 
-function helm_integration_tests() {
+helm_integration_tests() {
     helm_path=$bindir/helm
-    helm_chart="$( cd $bindir/.. && pwd )"/charts/linkerd2
+    helm_chart="$( cd "$bindir"/.. && pwd )"/charts/linkerd2
     helm_release_name=$linkerd_namespace-test
     tiller_namespace=$linkerd_namespace-tiller
 
     run_helm_test
-    exit_on_err "error testing Helm"
+    exit_on_err 'error testing Helm'
     helm_cleanup
-    exit_on_err "error cleaning up Helm"
+    exit_on_err 'error cleaning up Helm'
     # clean the data plane test resources
     cleanup
 }
 
-function deep_integration_tests() {
+deep_integration_tests() {
     run_test "$test_directory/install_test.go" --linkerd-namespace=$linkerd_namespace
-    exit_on_err "error during install"
+    exit_on_err 'error during install'
 
     run_test "$(go list $test_directory/.../...)" --linkerd-namespace=$linkerd_namespace
-    exit_on_err "error during deep tests"
+    exit_on_err 'error during deep tests'
 }
 
 #
 # Helper functions.
 #
 
-function check_linkerd_binary(){
-    printf "Checking the linkerd binary..."
+check_linkerd_binary(){
+    printf 'Checking the linkerd binary...'
     if [[ "$linkerd_path" != /* ]]; then
-        printf "\\n[%s] is not an absolute path\\n" "$linkerd_path"
+        printf '\\n[%s] is not an absolute path\\n' "$linkerd_path"
         exit 1
     fi
     if [ ! -x "$linkerd_path" ]; then
-        printf "\\n[%s] does not exist or is not executable\\n" "$linkerd_path"
+        printf '\\n[%s] does not exist or is not executable\\n' "$linkerd_path"
         exit 1
     fi
     exit_code=0
     "$linkerd_path" version --client > /dev/null 2>&1
-    exit_on_err "error running linkerd version command"
-    printf "[ok]\\n"
```

Comment:
Seems like changing from "" to '' messes up the new line  char interpretation and now this actually gets rendered as \n instead of producing a new line in my terminal.

Reply:
Should be printf '[ok]\n' and not printf '[ok]\\n'. Will fix.